tests/bdd/features/helm_chart/install.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Install Mayastor
+  Install Mayastor on a kubernetes cluster, using helm
+
+  Background:
+    Given: A Kubernetes cluster
+
+  Scenario: Mayastor helm chart install
+    Given worker nodes on the kubernetes cluster are labelled with the label 'openebs.io/engine=mayastor'
+    And a kubernetes namespace to install the Mayastor helm chart
+    When the Mayastor helm chart is installed with callhome disabled and at most 3 replicas of etcd
+    Then the helm chart release is in 'deployed' state
+    And one api-rest Kubernetes Deployment is Available
+    And one agent-core Kubernetes Deployment is Available
+    And one csi-controller Kubernetes Deployment is Available
+    And one operator-diskpool Kubernetes Deployment is Available
+    And one localpv-provisioner Kubernetes Deployment is Available
+    And one csi-node Kubernetes DaemonSet for which all replicas are Available
+    And one ha-node Kubernetes DaemonSet for which all replicas are Available
+    And one io-engine Kubernetes DaemonSet for which all replicas are Available
+    And one promtail Kubernetes DaemonSet for which all replicas are Available
+    And one etcd Kubernetes StatefulSet for which all replicas are Available
+    And one loki Kubernetes StatefulSet for which all replicas are Available
+    And one nats Kubernetes StatefulSet for which all replicas are Available
```

Comment:
I think you should use `Scenario Outline` and `Examples` here

Reply:
Yeah. We should use `Scenario Outline` for these kind of test case.

Reply:
I've found it difficult to implement the feature file using 'Scenario Outline' and 'Examples'. I'd like some help in this matter.

Reply:
Here's an example for now: https://github.com/openebs/mayastor-control-plane/blob/c52493d5618a758b3aac6668e3d90d5f043a863d/tests/bdd/features/csi/node/parameters.feature#L7
We can sync up on monday

Reply:
Alright, thanks.

Reply:
Resolving this conversation, because this test is no longer a part of this PR.